### PR TITLE
New enterprise structure [wip]

### PR DESCRIPTION
**Pull request by ryancreatescopy**
## Description

Our Enterprise content needs to accommodate information on mainnet, not just private blockchains. This PR adds new content to the site that talks about the pros of using mainnet for your business and reorganises the structure so that users can browse information on mainnet Ethereum and private blockchains from an enterprise perspective.

cc @tasdienes

### CODE REVIEW

**Review by tasdienes — DISMISSED** [review]
I tweaked a couple of links. Looks good to go now.

**Comment by ryancreatescopy** [comment]
@samajammin I propose we merge this and add Enterprise to the footer as part of #1738

**Comment by wackerow** [comment]
fyi- Latest commits for #1738 have `Enterprise` as it's own category in the footer, currently arranged last (if available for region)

**Review by tasdienes — DISMISSED** [review]

**Review by tasdienes — DISMISSED** [review]

**Review by tasdienes — DISMISSED** [review]

**Review by tasdienes — DISMISSED** [review]

**Review by tasdienes — DISMISSED** [review]

File: src/content/enterprise/private-ethereum/index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Private Ethereum for Enterprise
+description: Resources for enterprise applications on private Ethereum blockchains
+lang: en
+sidebar: true
+---
+
+# Private Ethereum for Enterprise {#private-ethereum-for-enterprise}
+
+Enterprise blockchain applications can be built on the public permissionless Ethereum Mainnet, or on private blockchains that are based on Ethereum technology. For more information on building on the public Ethereum Mainnet, see ethereum.org/enterprise.
```
Comment (samajammin) [inline src/content/enterprise/private-ethereum/index.md:10]:
```suggestion
Enterprise blockchain applications can be built on the public permissionless Ethereum Mainnet, or on private blockchains that are based on Ethereum technology. For more information on building on the public Ethereum Mainnet, see [Ethereum Mainnet for Enterprise](/enterprise/).
```

File: src/components/Nav/index.js [hunk 2]
```diff
@@ -265,7 +265,24 @@ const Nav = ({ handleThemeChange, isDarkTheme, path }) => {
     {
       text: "page-enterprise",
       to: "/enterprise/",
-      shouldDisplay: contentVersion > 1,
+      shouldDisplay: contentVersion == 1,
```
Comment (samajammin) [inline src/components/Nav/index.js:268]:
```suggestion
      shouldDisplay: contentVersion === 1.1,
```

File: src/intl/en.json [hunk 3]
```diff
@@ -78,6 +78,9 @@
   "page-developers-aria-label": "Developers' Menu",
   "page-edn-learn": "Learn",
   "page-enterprise": "Enterprise",
+  "page-enterprise-aria-label": "Enterprise's Menu",
```
Comment (samajammin) [inline src/intl/en.json:81]:
```suggestion
  "page-enterprise-aria-label": "Enterprise Menu",
```

**Review by samajammin — DISMISSED** [review]
Looks great - thanks @tasdienes @ryancreatescopy!

Made a couple edits, will approve & merge.

**Review by samajammin — APPROVED** [review]

**Pull request merged**